lib/helper/WebDriverIO.js

```diff
@@ -724,6 +765,26 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Return attribute value from random element, if there is several elements with the same locator
+   */
+  grabAttributeFromRandomElement(locator, attr) {
```

Comment:
what is the purpose of that? 0.o
I don't see test should rely on something random! 
I'm **completely against this method**.
Please rethink the implementation. You can use *random* in your own test but we should not provide this in core

**Remove this method**